libevmasm/RuleList.h

```diff
@@ -46,6 +46,11 @@ template <class S> S modWorkaround(S const& _a, S const& _b)
 	return (S)(bigint(_a) % bigint(_b));
 }
 
+template <class S> S minOfu256Add(S const& _a, S const& _b, S const& _c)
```

Comment:
Please move to `libdevcore/CommonData.h` after `bigintShiftLeftWorkaround`.

Reply:
Are you sure that this makes a nice helper function? It's rather cryptic - I don't think it's apparent what it does without looking at the implementation. And it's not like we're working around a bug in boost here or something like that...

Reply:
I also think it should just be inlined.